packages/fast-components-react-base/src/listbox/listbox.tsx

```diff
@@ -229,7 +294,16 @@ class Listbox extends Foundation<
      * Create class names
      */
     protected generateClassNames(): string {
-        return super.generateClassNames(get(this.props.managedClasses, "listbox", ""));
+        let className: string = get(this.props.managedClasses, "listbox", "");
+
+        if (this.props.disabled) {
+            className = className.concat(
+                " ",
+                get(this.props.managedClasses, "listbox__disabled")
```

Comment:
please check that this managedClass exists before attempting to concat.

Reply:
Doing our more standard approach instead: 
       if (this.props.disabled) {
            className = `${className} ${get(
                this.props,
                "managedClasses.listbox__disabled",
                ""
            )}`;
        }